src/pages/[[...slug]].tsx

```diff
@@ -287,6 +287,7 @@ export async function getStaticProps(context: GetStaticPropsContext) {
           resource,
           bannerData,
           headerFooterData,
+          revalidate: 30,
```

Comment:
These are the props passed into the `<ResourcePage>` component.

I think you mean for this to be [sibling to the `props` property](https://nextjs.org/docs/pages/building-your-application/data-fetching/incremental-static-regeneration#on-demand-revalidation-with-revalidatepath)?